packages/jaeger-ui/src/tracing.jsx

```diff
@@ -0,0 +1,51 @@
+import { Resource } from '@opentelemetry/resources';
+
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+
+import { WebTracerProvider, SimpleSpanProcessor, ConsoleSpanExporter } from '@opentelemetry/sdk-trace-web';
+
+import { OTLPTraceExporter }  from '@opentelemetry/exporter-trace-otlp-http';
+
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+
+import { FetchInstrumentation } from '@opentelemetry/instrumentation-fetch';
+
+const consoleExporter = new ConsoleSpanExporter();
+
+const collectorExporter = new OTLPTraceExporter({
+  url: `${process.env.OTEL_HOST}:4318/v1/traces`,
```

Comment:
I don't think this is how we want to do that. The binary that runs Jaeger UI (all-in-one or jaeger-query) has an internal connectivity to Jaeger backend and knows where to send the traces (since it's using OTEL SDK internally it can be configured in a standard way). Establishing a direct connection from the browser requires additional configuration, CORS settings, etc. I don't know what OTEL JS recommends here, but if I were designing it I would create an endpoint in the query API (which the UI already talks to) and submit the spans there. Then the endpoint could act as a proxy and forward the spans to the collector location.

Reply:
Opentelemetry tells to add tracing imports to index.html like 
```html
<script type="module" src="tracing.ts"></script>
```

> Establishing a direct connection from the browser requires additional configuration, CORS settings, etc.

Otel asks to add this in collector config

```yaml
receivers:
  otlp:
    protocols:
      http:
        cors:
          allowed_origins: ["*"] #required urls
          allowed_headers: ["*"] #required headers
```


Going through Otel docs and also the API way as suggested.